ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -25,6 +25,13 @@ protected void configure() {
 		Set<String> servers;
 		String clientId;
 		Long maxBlockMillis = TimeUnit.MINUTES.toMillis(1);
+		boolean enabled = true;
```

Comment:
Curious if we want to set defaults or just fall back to the driver defaults if nothing is specified.  Feels like using driver defaults might be the better way to go to me? Thoughts?